site/TombstonePage.tsx

```diff
@@ -43,10 +51,11 @@ export default function TombstonePage({
                         <p className="body-3-medium">
                             {reason || DEFAULT_TOMBSTONE_REASON}
                         </p>
-                        {!relatedLink && (
+                        {includeArchiveLink && (
                             <p className="body-3-medium">
                                 If you’d still like to view this page, you can
-                                find it{" "}
+                                find it via the Internet Archive's Wayback
+                                Machine{" "}
```

Comment:
Especially because this wraps in an unflattering way on desktop

<img width="624" alt="image" src="https://github.com/user-attachments/assets/9b027583-d9b8-4a78-947e-7ccef2afd8f4">

it's generally preferable to use more descriptive/explanatory text for anchor tags for screen readers (so that pressing tab doesn't just say the word "here")

e.g. 

> If you’d still like to view this page, you can check the [Internet Archive Wayback Machine](https://web.archive.org/web/*/http://localhost:3030/ikes-test-article).

Reply:
Good point, changed.